mitosheet/mitosheet/public/v3/sheet_functions/bool_functions.py

```diff
@@ -60,10 +59,73 @@ def AND(*argv: BoolInputType) -> BoolFunctionReturnType:
         lambda previous_series, new_series: previous_series & new_series
     )
 
+@cast_values_in_arg_to_type('series', 'bool')
+@handle_sheet_function_errors
+def BOOL(series: BoolRestrictedInputType) -> BoolFunctionReturnType:
+    """
+    {
+        "function": "BOOL",
+        "description": "Converts the passed arguments to boolean values, either True or False. For numberic values, 0 converts to False while all other values convert to True.",
+        "search_terms": ["bool", "boolean", "true", "false", "dtype", "convert"],
+        "examples": [
+            "BOOL(Amount_Payed)",
+            "AND(BOOL(Amount_Payed), Is_Paying)"
+        ],
+        "syntax": "BOOL(series)",
+        "syntax_elements": [{
+                "element": "series",
+                "description": "An series to convert to boolean values, either True or False."
+            }
+        ]
+    }
+    """
+    if isinstance(series, bool):
```

Comment:
One change we talked about in pair programming was updating these variable names from `series` to something like `series_or_primitive` or `bool_restricted_input_type` or something that makes it way more clear that its not necessarily a series.